lib/cusparse/conversions.jl

```diff
@@ -214,16 +225,24 @@ function CuSparseMatrixCSR{T}(S::Adjoint{T, <:CuSparseMatrixCSC{T}}) where {T <:
     return CuSparseMatrixCSR{T}(csc.colPtr, csc.rowVal, conj.(csc.nzVal), size(csc))
 end
 
-for SparseMatrixType in [:CuSparseMatrixCSC, :CuSparseMatrixCSR]
+for SparseMatrixType in (:(CuSparseMatrixCSC), :(CuSparseMatrixCSR), :(CuSparseMatrixCOO))
```

Comment:
There's no need to use parentheses here, just `:CuSparseMatrixCSC` works for plain symbols.

Reply:
```suggestion
for SparseMatrixType in (:CuSparseMatrixCSC, :CuSparseMatrixCSR, :CuSparseMatrixCOO)
```